### PR TITLE
fix(core): change styling tabs component

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
Closes SAP/fundamental-ngx#5128 _(original PR https://github.com/OleksiyDrozdov/fundamental-ngx/pull/38)_

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by kavya-b** [comment]
By default, the first tab is not appearing selected for all the examples that use `routerLink`. When we click on the tab, only then it gets selected. Maybe something needs to be changed with the actual routes path? I'm not sure.
Screenshot 2021-06-14 at 2 47 11 PM

**Comment by OleksiyDrozdov** [comment]
Screenshot 2021-06-14 145527
Issue was when you created a new project and imported fundamental-ngx you couldn't see selected link (there was no underlying) after user clicks the link but now it is fixed.

**Review by kavya-b — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

File: libs/core/src/lib/tabs/tab-nav/tab-nav.component.scss [hunk 1]
```diff
@@ -1 +1,18 @@
...
+        color: var(--fdTabs_Selected_Icon_Color, var(--sapContent_ContrastIconColor, #fff));
+        background-color: #0854a0;
+        background-color: var(--fdTabs_Selected_Color, var(--sapSelectedColor, #0854a0));
+
+        &::after {
+            background-color: #0854a0;
+            background-color: var(--fdTabs_Selected_Color, var(--sapSelectedColor, #0854a0));
+        }
+    }
+}
```
Comment (InnaAtanasova) [inline libs/core/src/lib/tabs/tab-nav/tab-nav.component.scss:18]: This needs to go in Fund-styles. We are no longer allowed to approve PRs with css that belongs in fund-styles.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

**Comment by OleksiyDrozdov** [comment]
https://github.com/OleksiyDrozdov/fundamental-styles/pull/1

**Pull request closed without merging**